templates/k8smetricsservicemonitor.go

```diff
@@ -52,8 +52,10 @@ var K8sMetricsServiceMonitorSpecTemplate = promv1.ServiceMonitorSpec{
 			},
 			TLSConfig: &promv1.TLSConfig{
 				SafeTLSConfig: promv1.SafeTLSConfig{
-					InsecureSkipVerify: ptr.To(true),
+					InsecureSkipVerify: ptr.To(false),
+					ServerName:         ptr.To("prometheus-k8s.openshift-monitoring.svc"),
```

Comment:
It wont be different in case of in house prometheus?

Reply:
It would be `(prometheus.%s.svc, namespace)` for the ROSA?

Reply:
It would be `(prometheus.%s.svc, namespace)` for the ROSA?

Reply:
In case of ROsa, `prometheus located in openshift-monitoring` wont fetch the metrics?

Reply:
Ack